modules/ROOT/pages/dataweave-system-properties.adoc

```diff
@@ -97,4 +97,6 @@ Use this property for troubleshooting, for example, if `CursorProvider#openCurso
 `com.mulesoft.dw.buffer.memory.monitoring` supports an _experimental feature_ that is subject to change or removal from future versions of DataWeave.
 |`com.mulesoft.dw.xml.supportDTD`|`Boolean`|`false`|Controls whether DTD handling is enabled or disabled. When this property is set to `false`, DataWeaves skips processing of both internal and external subsets. Note that the default for this property changed from `true` to `false` in Mule version 4.3.0-20210427, which includes the May, 2021 patch of DataWeave version 2.3.0.
 |`com.mulesoft.dw.multipart.defaultContentType`|`String`|`application/octet-stream`|Sets the default Content-Type to use on parts of the `multipart/*` format when a `Content-Type` is not specified. See also, the multipart reader property xref:dataweave-formats-multipart.adoc#properties[defaultContentType]. _Introduced in DataWeave 2.3 (2.3.0-20210720) for the August 2021 release of Mule 4.3.0-20210719._
+|`com.mulesoft.dw.TODOTODOTODO.useMissCharAsDefaultForFill`|`String`| TODO_true_or_false_or |When set to `false`, the flat file reader and writer use spaces for missing characters. When set to `true`, the operations uses TODO_WHAT_DOES_IT_DO. _Introduced in DataWeave 4.3 (2.3.0-20210823) for the September 2021 release of Mule 4.3.0-20210823._
```

Comment:
Need help with ALL TODOs  in the green.

Reply:
Is there a system property useMissCharAsDefaultForFill? Or is it just reader and writer properties for the flat file format?

Reply:
And if there is a system property, what is the full name?

Reply:
```suggestion
```